modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/DataRegionMetricsImpl.java

```diff
@@ -113,17 +114,21 @@
     /** Time interval (in milliseconds) when allocations/evictions are counted to calculate rate. */
     private volatile long rateTimeInterval;
 
+    /** Grid kernal context. */
+    private final GridKernalContext ctx;
+
     /**
      * @param memPlcCfg DataRegionConfiguration.
-     * @param mmgr Metrics manager.
+     * @param ctx Grid kernal context.
      * @param dataRegionMetricsProvider Data region metrics provider.
      */
     public DataRegionMetricsImpl(DataRegionConfiguration memPlcCfg,
-        GridMetricManager mmgr,
+        GridKernalContext ctx,
```

Comment:
Let's have 2 specific managers instead of `GridKernalContext` here.

Reply:
Please, just add `PerformanceStatisticaManager` to the constructor.

Reply:
fixed